hazelcast/src/main/java/com/hazelcast/client/api/Client.java

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-package com.hazelcast.core;
```

Comment:
I guess this is related to Java 9 package issues?

Reply:
Yes, `c.h.client` package is used by `hazelcast-client` module and cannot move these classes into `c.h.client` in core module. That's why `c.h.client.api` package..